config/rbac/role.yaml

```diff
@@ -29,6 +29,44 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - aadpodidentity.k8s.io
+  resources:
+  - azureassignedidentities
```

Comment:
`azureassignedidentities` are not required for managed mode

Reply:
will remove as well